src/main/java/picocli/CommandLine.java

```diff
@@ -15063,7 +15041,7 @@ private Text createLongOptionText(OptionSpec option, IParamLabelRenderer rendere
          *   PositionalParamSpec#description()} array, and these rows look like {@code {"", "", "", param.description()[i]}}.</p>
          */
         static class DefaultParameterRenderer implements IParameterRenderer {
-            private String requiredMarker = " ";
+            private String requiredMarker;
             private boolean showDefaultValues;
```

Comment:
Can we make `requiredMarker` and `showDefaultValues` `final` fields?

Reply:
Yes, I added a commit with some other private fields being made final.